extensions/core/deployment/src/main/java/io/quarkiverse/cxf/deployment/CxfBuildTimeConfig.java

```diff
@@ -445,4 +445,116 @@ default void validate(String prefix) {
         }
     }
 
+    /**
+     * Parameters for fully customizable algorithm suite.
+     */
+    @WithName("customizedAlgorithmSuite")
+    public CustAlgSuite custAlgSuite();
+
+    @ConfigGroup
+    public interface CustAlgSuite {
+
+        public static final String CUSTOMIZED_ALGORITHM_SUITE_NAME = "CustomizedAlgorithmSuite";
```

Comment:
```suggestion
        public static final String CUSTOMIZED_ALGORITHM_SUITE_NAME = "QuarkusCxfFipsAlgorithmSuite";
```

... so that users have a chance to see where it comes from when they see it in policy files or in some sort of a debug output.